components/brave_wallet/common/f4_jumble.cc

```diff
@@ -0,0 +1,198 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/common/f4_jumble.h"
+
+#include <algorithm>
+#include <array>
+#include <memory>
+#include <string>
+
+#include "base/notreached.h"
+#include "brave/components/brave_wallet/common/hex_utils.h"
+#include "brave/third_party/argon2/src/src/blake2/blake2.h"
+
+namespace brave_wallet {
+
+const size_t kLeftSize = 64u;
+
+std::string GetHPersonalizer(uint8_t i) {
+  return {85,  65, 95,  70,  52, 74, 117,
+          109, 98, 108, 101, 95, 72, static_cast<char>(i),
+          0,   0};
+}
+
+std::string GetGPersonalizer(uint8_t i, uint16_t j) {
+  return {85,
+          65,
+          95,
+          70,
+          52,
+          74,
+          117,
+          109,
+          98,
+          108,
+          101,
+          95,
+          71,
+          static_cast<char>(i),
+          static_cast<char>(j & 0xFF),
+          static_cast<char>(j >> 8)};
+}
+
```

Comment:
These should return std:array<uint8_t, 16>

Reply:
Done